.github/workflows/Nightly_Perf_Env_Even_CI.yml

```diff
@@ -6,7 +6,7 @@ name: Deploy Perf Env Nightly CI
 on:
   # Run on Nightly
   schedule:
-    - cron: '0 4 * * 0,2-6' # run Sunday, Tuesday-Sunday at 4 AM UTC/ 0 AM EDT
+    - cron: '0 4 * * 3,5' # run Sunday, Tuesday-Sunday at 4 AM UTC/ 0 AM EDT
```

Comment:
0 4 * * 1,3,5
run Monday, Wednesday, and Friday at 4 AM UTC

Reply:
Fix the comment, too.